ios/FluentUI/PersonaButton/PersonaButtonTokenSet.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined sizes of the persona button
+@objc public enum MSFPersonaButtonSize: Int, CaseIterable {
+    case small
+    case large
+
+    var avatarSize: MSFAvatarSize {
+        switch self {
+        case .large:
+            return .xxlarge
+        case .small:
+            return .xlarge
+        }
+    }
+
+    var shouldShowSubtitle: Bool {
+        switch self {
+        case .large:
+            return true
+        case .small:
+            return false
+        }
+    }
+}
+
+/// Representation of design tokens to persona buttons at runtime which interfaces with the Design Token System auto-generated code.
+/// Updating these properties causes the SwiftUI persona button to update its view automatically.
+public class PersonaButtonTokenSet: ControlTokenSet<PersonaButtonTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// The amount of space between the control's `Avatar` and text labels.
+        case avatarInterspace
+
+        /// The background color for the `PersonaButton`.
+        case backgroundColor
+
+        /// How much space should be reserved to the left and right of the control's `Avatar`.
+        case horizontalAvatarPadding
+
+        /// How much space should be reserved to the left and right of the control's labels.
+        case horizontalTextPadding
+
+        /// The `DynamicColor` to use for the control's primary label.
+        case labelColor
+
+        /// The `FontInfo` to use for the control's primary label.
+        case labelFont
+
+        /// The `DynamicColor` to use for the control's secondary label.
+        case sublabelColor
+
+        /// The `FontInfo` to use for the control's secondary label.
+        case sublabelFont
+
+        /// How much padding to add above the `Avatar` and below the lowest text label.
+        case verticalPadding
+    }
+
+    init(size: @escaping () -> MSFPersonaButtonSize) {
+        self.size = size
+        super.init()
+    }
+
+    @available(*, unavailable)
+    required init() {
+        preconditionFailure("init() has not been implemented")
+    }
+
+    override func defaultValue(_ token: Tokens) -> ControlTokenValue {
+        switch token {
+            /// The amount of space between the control's `Avatar` and text labels.
+        case .avatarInterspace:
+            return .float {
```

Comment:
The spacing looks a bit odd. Should the comment be further in than the `case`?

Reply:
Yep probably :(

Reply:
Strangely, this is actually how Xcode indents this automatically. That said, I don't mean for those comments to be there anyway (they should only be up where the `case`s are defined), so I'm just removing them.